Dockerfile-dev

```diff
@@ -0,0 +1,26 @@
+#
+# Dockerfile for an image with the currently checked out version of zipline installed. To build:
+#
+#    docker build -t quantopian/ziplinedev -f quantopian/ziplinedev .
```

Comment:
Sure I'll get it done today

Sent from my iPhone

> On Jun 6, 2016, at 4:29 PM, Richard Frank <notifications@github.com> wrote:
> 
> In Dockerfile-dev:
> 
> > @@ -0,0 +1,26 @@
> > +#
> > +# Dockerfile for an image with the currently checked out version of zipline installed. To build:
> > +#
> > +#    docker build -t quantopian/ziplinedev -f quantopian/ziplinedev .
> Almost there...
> 
> $ docker build -t quantopian/ziplinedev -f quantopian/ziplinedev .
> Cannot locate Dockerfile: quantopian/ziplinedev
> Looks like we're using the image name instead of the file name:
> 
> $ docker build -t quantopian/ziplinedev -f ./Dockerfile-dev .
> Sending build context to Docker daemon 
> Step 0 : FROM python:2.7
> ...
> Mind just fixing that up, and we'll get this merged? Thanks so much!
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>